.circleci/config.yml

```diff
@@ -489,7 +489,8 @@ jobs:
       name: win/default
       shell: bash.exe
     steps:
-      - run: choco install python --version=3.10.4
+      # circleci/windows@5.0 orb includes python 3.10.6
+      # - run: choco install python --version=3.10.4
```

Comment:
```suggestion
```

Reply:
Thanks for flagging this!

This install failed with an error:

```
WARNING: Generic MSI Error. This is a local environment error, not an issue with a package or the MSI itself - it could mean a pending reboot is necessary prior to install or something else (like the same version is already installed). Please see MSI log if available. If not, try again adding '--install-arguments="'/l*v c:\python3_msi_install.log'"'. Then search the MSI Log for "Return Value 3" and look above that for the error.
```

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/21831/workflows/2816c922-129e-414c-993d-27020e081da5/jobs/1484574

This can also be taken up in a follow-up PR where these profiler windows jobs are reviewed with the updated windows orb.